tools/clang/test/CodeGenSPIRV/intrinsics.normalize.hlsl

```diff
@@ -0,0 +1,32 @@
+// Run: %dxc -T vs_6_0 -E main
+
+// According to HLSL reference:
+// The 'normalize' function can only operate on floats and vector of floats.
```

Comment:
same thing here...
I think normalize(X) = X where X is a scalar. But the spec doesn't specify the behavior, so I'll remove this test case.